src/Sylius/Behat/Context/Ui/Admin/ManagingProductsContext.php

```diff
@@ -662,6 +663,39 @@ public function iAttachImageWithoutACode($path)
     }
 
     /**
+     * @When I associate as :productAssociationType the :productName product
+     * @When I associate as :productAssociationType the :firstProductName and :secondProductName products
+     */
+    public function iAssociateProductsAsProductAssociation(
+        ProductAssociationTypeInterface $productAssociationType,
+        ...$productsNames
+    ) {
+        /** @var UpdateSimpleProductPageInterface|UpdateConfigurableProductPageInterface $currentPage */
+        $currentPage = $this->currentPageResolver->getCurrentPageWithForm([
+            $this->createSimpleProductPage,
```

Comment:
I don't think that testing that also on configurable product page has a business value, because the logic is the same.